loopslab.js

```diff
@@ -0,0 +1,64 @@
+// Number 1a Write a while loop and a for loop that takes the variable "num" and logs all the numbers,
```

Comment:
This is incomplete for now so please continue working on your assignment. Even though it's late submission, it's more important to understand concepts sufficiently which I believe you will. Continuously reach out to me and Alejo for any questions. Keep it up!